tests/test_series.py

```diff
@@ -781,3 +782,15 @@ def test_iloc_setitem_ndarray() -> None:
     values_s.iloc[indices_u16] = -1
     values_s.iloc[indices_u32] = -1
     values_s.iloc[indices_u64] = -1
+
+
+def test_categorical_codes():
+    # GH-111
+    codes: Union[
+        npt.NDArray[np.int8],
+        npt.NDArray[np.int16],
+        npt.NDArray[np.int32],
+        npt.NDArray[np.int64],
+    ]
+    cat = pd.Categorical(["a", "b", "a"])
+    codes = cat.codes
```

Comment:
Can you do this instead?
1.  import `np_ndarray_int` as follows at the top:
```python
if TYPE_CHECKING:
    from pandas._typing import np_ndarray_int
```
2. Then  you can remove the type of `codes` in the test, and just do:
```python
assert_type(cat.codes, "np_ndarray_int")
```